libraries/xcode.rb

```diff
@@ -2,10 +2,24 @@ module Xcode
   module Helper
     BASE_COMMAND ||= '/usr/local/bin/xcversion'.freeze
```

Comment:
We should extract this to a method instead:

```ruby
def xcversion_command
  '/usr/local/bin/xcversion'.freeze
end

Reply:
Done.